projects/plugins/jetpack/extensions/blocks/podcast-player/edit.js

```diff
@@ -154,8 +155,12 @@ const PodcastPlayerEdit = ( {
 
 		// Clean current podcast feed and fetch a new one.
 		setFeedData( {} );
-		fetchFeed( url );
-	}, [ fetchFeed, removeAllNotices, url ] );
+		fetchFeed(
+			url,
+			selectedEpisodes.map( episode => episode.guid )
```

Comment:
why do you pass `guids` as a second parameter instead of being another key of the request feed? Some lines above, the request is performed by the fetch helper:

```es6
fetchPodcastFeed( { url: urlToFetch, guids } )
```

where it has only one parameter which is an object: `{ url: urlToFetch, guids }`

It's inconsistent with the `fetchFeed()`, accepting two parameters instead of an object.

I suggest something like...
```es6
fetchFeed( {
	url,
	guids: selectedEpisodes.map( episode => episode.guid ),
} );
```

```es6
const fetchFeed = useCallback( ( requestParams = {} ) => {
	cancellableFetch.current = makeCancellable( fetchPodcastFeed( requestParams );
```

Reply:
I think this is probably because of how I implemented it in #18201. It's probably better to pass an object like this.